Source/Core/Core/HW/EXI/BBA/TAP_Unix.cpp

```diff
@@ -50,7 +52,7 @@ bool CEXIETHERNET::TAPNetworkInterface::Activate()
   const int MAX_INTERFACES = 32;
   for (int i = 0; i < MAX_INTERFACES; ++i)
   {
-    strncpy(ifr.ifr_name, StringFromFormat("Dolphin%d", i).c_str(), IFNAMSIZ);
+    strncpy(ifr.ifr_name, fmt::format("Dolphin{}", i).c_str(), IFNAMSIZ);
```

Comment:
`format_to` and `format_to_n` never write a null terminator, and I'm not sure if one is needed in this case, so it seemed safer to stick with the original implementation. (Though `strncpy` also does not write a null terminator if the original string is longer than the buffer size.)

Reply:
It's probably fine, `ifr` is zero'd before doing this. Just an observation, since other places do use it; feel free to leave it like that if nobody else has an opinion on it.

Reply:
I didn't see that earlier zeroing out; I've changed it to `format_to_n`.